package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backstage-plugin-techdocs-addon-mermaid",
-  "version": "0.12.1",
+  "version": "0.13.0",
```

Comment:
Could you revert this change? The version is generally only increased once the PR has been merged into main.

Reply:
@johanneswuerbach Sorry about that. Thanks for your patience.